src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,72 @@
+package command;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Map;
+
+import exception.DescriptionException;
+import exception.DukeDateTimeParserException;
+import parser.Parser;
+import storage.Storage;
+import task.DeadlineTask;
+import task.TaskList;
+import ui.Ui;
+
+
+
+/**
+ * DeadlineCommand class would execute the program when user specify
+ * "deadline" in the command. It would parse the command and show
+ * message to user, as well as updating the external file.
```

Comment:
Javadocs for Command sub-classes could be more concise as comments on `execute` already include some information.